### ISSUE TITLE
Not compatible with latest SlmQueue

### ISSUE THREAD

**Issue by joegreen88-liquid** [juriansluiman/SlmQueueBeanstalkd#55]
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package slm/queue (locked at 0.4.2, required as 0.6.*) is satisfiable by slm/queue[0.4.2] but these conflict with your requirements or minimum-stability.
  Problem 2
    - The requested package aws/aws-sdk-php-zf2 (locked at 2.0.2, required as 3.*) is satisfiable by aws/aws-sdk-php-zf2[2.0.2] but these conflict with your requirements or minimum-stability.
  Problem 3
    - slm/queue-beanstalkd v0.4.2 requires slm/queue 0.4.* -> satisfiable by slm/queue[0.4.0, 0.4.0-beta1, 0.4.0-beta2, 0.4.0-beta3, 0.4.1, 0.4.2] but these conflict with your requirements or minimum-stability.
    - slm/queue-beanstalkd v0.4.2 requires slm/queue 0.4.* -> satisfiable by slm/queue[0.4.0, 0.4.0-beta1, 0.4.0-beta2, 0.4.0-beta3, 0.4.1, 0.4.2] but these conflict with your requirements or minimum-stability.
    - slm/queue-beanstalkd v0.4.2 requires slm/queue 0.4.* -> satisfiable by slm/queue[0.4.0, 0.4.0-beta1, 0.4.0-beta2, 0.4.0-beta3, 0.4.1, 0.4.2] but these conflict with your requirements or minimum-stability.
    - Installation request for slm/queue-beanstalkd (locked at v0.4.2, required as 0.4.*) -> satisfiable by slm/queue-beanstalkd[v0.4.2].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by conorgahan** [comment]
Is there any reason why this has not been addressed to also allow SlmQueueBeanstalkd leverage the bug fix for the InterruptStrategy in SlmQueue v0.5.1?

**Comment by vitorbrandao** [comment]
https://github.com/juriansluiman/SlmQueueBeanstalkd/pull/63 was created to support `slm/queue v1.0+`
